README.md

```diff
@@ -54,29 +56,37 @@ It can:
 Only one address can have this role.
 
 It can:
-- Do what the allocators can do.
-- [Timelocked] Enable or disable a market by setting a cap to a specific market.
-    - The cap must be set to 0 to disable the market.
-	- Disabling a market can then only be done if the vault has no liquidity supplied on the market.
+
+- Do what allocators can do.
+- [Timelocked] Enable or disable a market by setting a supply cap to a specific market.
+  - The supply cap must be set to 0 to disable the market.
+  - Disabling a market can then only be done if the vault has no liquidity supplied on the market.
+- Revoke the pending cap of any market.
 
 #### Allocator
 
 Multiple addresses can have this role.
 
 It can:
+
 - Set the `supplyQueue` and `withdrawQueue`, i.e. decide on the order of the markets to supply/withdraw from.
-    - Upon a deposit, the vault will supply up to the cap of each Morpho Blue market in the supply queue in the order set. The remaining funds are left as idle supply on the vault (uncapped).
-	- Upon a withdrawal, the vault will first withdraw from the idle supply and then withdraw up to the liquidity of each Morpho Blue market in the withdrawal queue in the order set.
-	- The `supplyQueue` contains only enabled markets (enabled market are markets with non-zero cap or with non-zero vault's supply).
-	- The `withdrawQueue` contains all enabled markets.
-- Instantaneously reallocate funds among the enabled market at any moment.
+  - Upon a deposit, the vault will supply up to the cap of each Morpho Blue market in the supply queue in the order set. The remaining funds are left as idle supply on the vault (uncapped).
+  - Upon a withdrawal, the vault will first withdraw from the idle supply and then withdraw up to the liquidity of each Morpho Blue market in the withdrawal queue in the order set.
+  - The `supplyQueue` contains only enabled markets (enabled market are markets with non-zero cap or with non-zero vault's supply).
```

Comment:
This is not true because a market supply cap can be set to non-zero, that market is added to the supply queue, then the supply cap can be put to zero again (before anyone has had the chance to supply on it) but it would not be removed from the supply queue. One invariant we have though is that markets from the supply queue have been previously approved (set with non-zero supply cap) by the curator/owner

Reply:
Also the line "Instantaneously reallocate funds across enabled markets at any moment" is not accurate

Reply:
What's the problem with "Instantaneously reallocate funds across enabled markets at any moment"?